integrations/llama_index/README.md

```diff
@@ -0,0 +1,184 @@
+from dotenv import load_dotenv
+from llama_index.agent.openai import OpenAIAgent
+from llama_index.llms.openai import OpenAI
+from llama_index.core.tools import FunctionTool
+from skyvern_llamaindex.agent import SkyvernTaskToolSpec
+
+async def sleep(seconds: int) -> str:
+    await asyncio.sleep(seconds)
+    return f"Slept for {seconds} seconds"
+
+# load OpenAI API key from .env
+load_dotenv()
+
+skyvern_task_tool = SkyvernTaskToolSpec()
+
+sleep_tool = FunctionTool.from_defaults(
+    async_fn=sleep,
+    description="Sleep for a given number of seconds",
+    name="sleep",
+)
+
+tools = skyvern_task_tool.to_tool_list(["dispatch", "get"])
+tools.append(sleep_tool)
+
+agent = OpenAIAgent.from_tools(
+    tools=tools,
+    llm=OpenAI(model="gpt-4o"),
+    verbose=True,
+    max_function_calls=10,
+)
+
+response = agent.chat("Run a task with Skyvern. The task is about 'Navigate to the Hacker News homepage and get the top 3 posts.' Then, get this task information until it's completed. The task information re-get interval should be 60s.")
+print(response)
+
+```
+
+### Run a task(sync) with skyvern client (calling skyvern OpenAPI in the tool)
+> sync task won't return until the task is finished.
+
+no need to run `skyvern init` command in your terminal to set up skyvern before using this integration.
+
+```python
+import asyncio
+from dotenv import load_dotenv
+from llama_index.agent.openai import OpenAIAgent
+from llama_index.llms.openai import OpenAI
+from skyvern_llamaindex.client import SkyvernTaskToolSpec
+
+
+async def sleep(seconds: int) -> str:
+    await asyncio.sleep(seconds)
+    return f"Slept for {seconds} seconds"
+
+# load OpenAI API key from .env
+load_dotenv()
+
+skyvern_task_tool = SkyvernTaskToolSpec(
+    credential="<your_organization_api_key>",
+)
+
+tools = skyvern_task_tool.to_tool_list(["run"])
+
+agent = OpenAIAgent.from_tools(
+    tools=tools,
+    llm=OpenAI(model="gpt-4o"),
+    verbose=True,
+    max_function_calls=10,
+)
+
+response = agent.chat("Run the task with skyvern. The task is about 'Navigate to the Hacker News homepage and get the top 3 posts.'")
+print(response)
+
+```
+
+### Dispatch a task(async) with skyvern client (calling skyvern OpenAPI in the tool)
+> dispatch task will return immediately and the task will be running in the background. You can use `get` tool to poll the task information until the task is finished.
+
+no need to run `skyvern init` command in your terminal to set up skyvern before using this integration.
+
+```python
+import asyncio
+from dotenv import load_dotenv
+from llama_index.agent.openai import OpenAIAgent
+from llama_index.llms.openai import OpenAI
+from llama_index.core.tools import FunctionTool
+from skyvern_llamaindex.client import SkyvernTaskToolSpec
+
+
+async def sleep(seconds: int) -> str:
+    await asyncio.sleep(seconds)
+    return f"Slept for {seconds} seconds"
+
+# load OpenAI API key from .env
+load_dotenv()
+
+skyvern_task_tool = SkyvernTaskToolSpec(
```

Comment:
after using it and thinking abt it, let's just use one name/tool, named `SkyvernTool`.

For setting up workflow, we can still have different interfaces like `skyvern_tool.run_task` and `skyvern_tool.run_workflow`?